vpn/src/main/java/com/duckduckgo/mobile/android/vpn/processor/udp/UdpPacketProcessor.kt

```diff
@@ -242,6 +242,7 @@ private inline fun udpRunnable(crossinline block: () -> Unit): Runnable {
                 Timber.w(e, "Failed to process UDP network-to-device packet")
             } catch (e: InterruptedException) {
                 Timber.v("Thread interrupted")
+                break
```

Comment:
nit: would `return@Runnable` be equivalent/better

Reply:
Good call! Updated :+1: